salt-config/roster

```diff
@@ -33,6 +33,9 @@ cove-360-live:          cove-360-live.live.threesixtygiving.uk0.bigv.io
 data-360-live:          data.live.threesixtygiving.uk0.bigv.io
 datastore-360-live: datastore-dev.grantnav.threesixtygiving.org
 
+# Publish What You Fund (PWYF)
```

Comment:
Maybe a comment explaining this is a VM? Or to make it even easier for the people who do server updates every week, could this be commented out in salt-config/rooster with a comment explaining comment back in to use on a VM? Then they won't have to change their workflow at all and they won't see errors

Reply:
Yeah, I think what I'll do is just not commit an entry until we have an actual server to do it against, which hopefully will be soon